rest-api/alembic/versions/8786f0a95801_org_metadata.py

```diff
@@ -0,0 +1,84 @@
+"""Org metadata
+
+Revision ID: 8786f0a95801
+Revises: b315fec9aa4e
+Create Date: 2017-11-29 16:20:00.427120
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import model.utils
+from sqlalchemy.dialects import mysql
+
+from participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from participant_enums import WithdrawalStatus, SuspensionStatus
+from participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType
+from model.site_enums import SiteStatus
+from model.code import CodeType
+
+# revision identifiers, used by Alembic.
+revision = '8786f0a95801'
+down_revision = 'b315fec9aa4e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('organization',
+    sa.Column('organization_id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=80), nullable=False),
```

Comment:
Per comment I just left on the doc, I find this column name to be confusing in combination with "display_name".

ALso, is there generally any place in the code where documentation of these columns would live? Would be nice especially for columns like this one.

Reply:
I renamed 'name' to 'external_id'. Does that work?

Note that on hpo this is called 'name', and on site 'googleGroup', so we're not being very consistent... but changing either of those is a bigger deal, and should be tackled separate from this PR.

I added comments to organization.py on the fields. That's the best place we have for documenting model objects.

Reply:
Thanks, I find this to be much clearer.